File: serializacion/Serializacion.dbase/Modelo/Tarea.vb
<Serializable()> _
Public Class Tarea

    Private _nombre As String
    Private _descripcion As String
    Private _fechaFin As Date

    Public Sub New(ByVal pNombre As String, ByVal pDescripcion As String, ByVal pFechaFin As Date)

        _nombre = pNombre
        _descripcion = pDescripcion
        _fechaFin = pFechaFin

    End Sub

    Public Property Nombre() As String
        Get
            Return _nombre
        End Get
        Set(ByVal value As String)
            _nombre = value
        End Set
    End Property

    Public Property Descripcion() As String
        Get
            Return _descripcion
        End Get
        Set(ByVal value As String)
            _descripcion = value
        End Set
    End Property

    Public Property FechaFin() As Date
        Get
            Return _fechaFin
        End Get
        Set(ByVal value As Date)
            _fechaFin = value
        End Set
    End Property



End Class

File: serializacion/Serializacion.dbase/Model/Tarea.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model
{
    [Serializable]
    public class Tarea
    {
        private string _nombre;

        private string _descripcion;

        private DateTime _fechaFin;

        public Tarea(string pNombre, string pDescripcion, DateTime pFechaFin)
        {
            _nombre = pNombre;
            _descripcion = pDescripcion;
            _fechaFin = pFechaFin;
        }

        public string Nombre
        {
            get
            {
                return _nombre;
            }
            set
            {
                _nombre = value;
            }
        }

        public string Descripcion
        {
            get
            {
                return _descripcion;
            }
            set
            {
                _descripcion = value;
            }
        }

        public DateTime FechaFin
        {
            get
            {
                return _fechaFin;
            }
            set
            {
                _fechaFin = value;
            }
        }
    }
}
